### PR TITLE
Update pylint

### PR DESCRIPTION
**Pull request by jzoldak**
Travis was failing because of version conflicts among some combination pylint, astroid, and edx-lint.

@benpatterson 
Thx for the help @nedbat

### CODE REVIEW

**Comment by nedbat** [comment]
@jzoldak I thought you would add `disable=django-not-available` to the tweaks?

File: pylintrc [hunk 1]
```diff
@@ -96,7 +93,7 @@ ignore-imports = no
 [TYPECHECK]
 ignore-mixin-members = yes
 ignored-classes = SQLObject
-zope = no
+unsafe-load-any-extension = yes
```
Comment (benpatterson) [inline pylintrc:96]: what does this do?

**Comment by benpatterson** [comment]
I have a question just out of curiosity but :+1:

**Comment by jzoldak** [comment]
@nedbat I overrode `load-plugins = edx_lint.pylint` so that the django plugin didn't even need to load, instead.

File: pylintrc (hunk 1 above)
Comment (jzoldak) [inline pylintrc:96]: This came through with the new version of edx-lint. [Here's the reference.](https://github.com/edx/edx-lint/blob/faba6920bcbbc91a1064d3071723e095197b190f/edx_lint/files/pylintrc#L199)

**Comment by nedbat** [comment]
@jzoldak the reason I don't like the load-plugins override is that it clobbers the line.  if edx-lint adds more plugins in the future, this repo won't get them.

**Pull request merged**